x-pack/plugins/uptime/public/components/monitor/ping_list/columns/ping_timestamp.tsx

```diff
@@ -96,10 +96,7 @@ export const PingTimestamp = ({ timestamp, ping }: Props) => {
 
   const ImageCaption = (
     <>
-      <div
-        className="stepArrowsFullScreen"
-        style={{ position: 'absolute', bottom: 32, width: '100%' }}
-      >
+      <div className="stepArrowsFullScreen">
```

Comment:
Was it intentional to show the caption within the image originally?

Reply:
i don't think so. Not much thought was given to it probably originally.